root-project.gradle

```diff
@@ -100,7 +100,7 @@ configure(subprojects) {
     }
     task(['type': JacocoReport, 'dependsOn': 'check', 'group': 'Coverage',
           'description': 'Generates JaCoCo unit test coverage reports.'], checkCoverage) {
-        def excludes = ['**/R.class', '**/R$*.class', '**/BuildConfig.*', '**/proto/**', '**Manifest*.*']
+        def excludes = ['**/R.class', '**/R$*.class', '**/BuildConfig.*', '**/proto/**', '**Manifest*.*', '**/AutoValue_*.class', '**/*_*Factory.class']
```

Comment:
I'm not sure about these changes. When looking at the coverage reports it will seem like these `TransportRuntime` changes are related to the drop, when it's actually different. Consider moving them into their own PR.